src/main/apiClient.ts

```diff
@@ -311,14 +311,17 @@ export default class ApiClient {
   }
 
   loginWithPassword(params: LoginWithPasswordParams): Promise<void> {
-    const saveCredentials = !isUndefined(params.saveCredentials) && params.saveCredentials
+    const storeCredentialsInBrowser = !isUndefined(params.saveCredentials) && params.saveCredentials
 
     const loginPromise =
-      window.cordova || saveCredentials
+      window.cordova || storeCredentialsInBrowser
```

Comment:
Why did you change `saveCredentials ` into `storeCredentialsInBrowser`? It's different now from the parameter's name.

Reply:
It's about documenting what this param aims for. I can also add a comment and rollback its name.

Reply:
I fear it might be confusing to have 2 different variable names for the same value :/ I'd rather have a comment in the code (the parameter is already documented https://developer.reachfive.com/sdk-core/loginWithPassword.html#saveCredentials).

Reply:
Updated as you required